lib/integration/nation_builder.rb

```diff
@@ -62,6 +81,21 @@ def self.create_rsvp(event_id:, person_id:)
     end
   end
 
+  def self.create_event(attributes:)
+    rescue_oauth_errors do
+      body = {'event': parse_event_attributes(attributes)}
+      response = request_handler(endpoint_path: ENDPOINTS[:events], body: body, method: 'post')
+      response['event'].try(:fetch, 'id')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.